services/spar/src/Spar/App.hs

```diff
@@ -281,7 +282,7 @@ catchVerdictErrors = (`catchError` hndlr)
         Left (serr :: ServantErr) -> VerifyHandlerError "unknown-error" (cs (errReasonPhrase serr) <> " " <> cs (errBody serr))
 
 verdictHandlerResult :: HasCallStack => Maybe BindCookie -> SAML.AccessVerdict -> Spar VerdictHandlerResult
-verdictHandlerResult bindCky = catchVerdictErrors . \case
+verdictHandlerResult bindCky = (>>= logVerdictHandlerResult) . catchVerdictErrors . \case
```

Comment:
pleeeeeeease don't use sectioned `>>=`; just rewrite this block at this point...